vignettes/dettl.Rmd

```diff
@@ -111,11 +111,11 @@ The process at a minimum needs code to define 3 functions one for each stage of
 
 ## Creating a new ETL process
 
-If you want to create a new process within an existing project this can be done by opening the project as an RStudio project and using `new_dettl()`. Note that the use of `withr::with_dir(tempdir(), ...)` in the example below is because this is running in a vignette and the code is an example. This is not necessary when using `dettl` in a real environment.
+If you want to create a new process within an existing project this can be done by opening the project as an RStudio project and using `dettl_new()`. Note that the use of `withr::with_dir(tempdir(), ...)` in the example below is because this is running in a vignette and the code is an example. This is not necessary when using `dettl` in a real environment.
 
 ```{r}
 withr::with_dir(tempdir(), 
-  import_name <- dettl::new_dettl("person_information")
+  import_name <- dettl::dettl_new("person_information")
```

Comment:
See the comment above about a `path` argument.  I also wonder how much sense it makes to be able to run this in a directory that does not contain a `dettl_config.yml` file (another separate ticket certainly, if you agree)

Reply:
I also prefer the form

```r
import_name <- withr::with_dir(tempdir(), 
  dettl::dettl_new("person_information")
)
```

because it's less ambiguous about scoping rules - I'd missed the assignment here on the first read

Reply:
I've added a PR for first point. There would be little reason to do `dettl_new` unless your working directory contained a config file.

I've updated the code re the second point, I didn't know you could drive `withr` like that.